experimental/README.md

```diff
@@ -42,6 +42,23 @@ Unlike the regular Docker binary, the experimental channels is built and updated
 
     This command downloads a test image and runs it in a container.
 
+### Get the Linux binary
+To download the latest experimental `docker` binary for Linux,
+use the following URLs:
+
+    https://experimental.docker.com/builds/Linux/i386/docker-latest
+
+    https://experimental.docker.com/builds/Linux/x86_64/docker-latest
+
+After downloading the appropriate binary, you can follow the instructions
+[here](https://docs.docker.com/installation/binaries/#get-the-docker-binary) to run the `docker` daemon.
+
+> **Note**
```

Comment:
Thanks for the update @ankushagarwal . Only use ordered list for a procedure. Also, well this need not be a note.

---> https://gist.github.com/moxiegirl/e1e3bd03b49d33b4eab9


To get the MD5 and SHA256 hashes, append `.md5` and `.sha256` respectively to the URLs.  Also, to get the compressed binaries, just append `.tgz` to the URLs, for example:

        https://experimental.docker.com/builds/Linux/i386/docker-latest.tgz